src/components/common/ModeratorList.tsx

```diff
@@ -1,16 +1,22 @@
 import React from "react";
-import { Text } from "native-base";
+import { Text, VStack } from "native-base";
+import { CommunityModeratorView } from "lemmy-js-client";
 import AvatarUsername from "./AvatarUsername";
 
 interface IProps {
-  item: any;
+  moderators: CommunityModeratorView[];
```

Comment:
I changed this to just pass the moderators list instead of being for 1 item in the list. Also don't use `any` if you can. It makes it so typescript just doesn't work, which is not actually helpful generally.